app/javascript/src/components/Dashboard/Notes/Pane/CreateNote.jsx

```diff
@@ -6,14 +6,18 @@ import formInitialValues from "constants/formInitialValues";
 
 import Form from "./Form";
 
-export default function NewNotePane({ fetchNotes, showPane, setShowPane }) {
-  const onClose = () => setShowPane(false);
+export default function NewNotePane({
```

Comment:
Why is this component called `CreateNote` but export function called `NewNotePane`? 
Won't it confuse other developers?

Reply:
True. In fact, we should not be naming files or components with `Pane, Modal, Form, Page` as suffixes or prefixes.